src/WikiSearch/WikiSearch.tsx

```diff
@@ -0,0 +1,177 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import { createRef, useEffect, useMemo, useState } from "react";
+import { DebouncedFunc, debounce, throttle } from "lodash";
+import Styles from "./WikiSearch.module.scss";
+
+import SearchIcon from "../Search.svg";
+
+import { Popover, PopoverContent, PopoverDescription, PopoverTrigger } from "../Popover/Popover";
+import { useSearchParams } from "react-router-dom";
+import OpenSearchQueryResponse from "./OpenSearchQueryResponse.type";
+
+const itemLimit = 100;
```

Comment:
better to put in a separate constant file where you put all the application constants in one place.